autopush/settings.py

```diff
@@ -163,9 +162,12 @@ def __init__(self,
         self.router = Router(self.router_table, self.metrics)
 
         # Used to determine whether a connection is out of date with current
-        # db objects
-        self.current_msg_month = make_rotating_tablename(self._message_prefix)
-        self.current_month = datetime.date.today().month
+        # db objects. There are three noteworty cases:
+        # 1 "Last Month" the table requires a rollover.
+        # 2 "This Month" the most common case.
+        # 3 "Tomorrow" in the case where the system is actively rolling over
+        #   and one node is not yet using the latest clock, no rollover
+        #   required.
```

Comment:
"Next Month", where the system will soon be rolling over, but with timing, some nodes may roll over sooner. Ensuring the next month's table is present before the switchover is the main reason for this, just in case some nodes do switch sooner.